dotnet/Capstone/Controllers/BirdSightingController.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Capstone.DAO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System;
+using Capstone.Models;
+using Capstone.Exceptions;
+using System.Security.Principal;
+
+namespace Capstone.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class BirdSightingController : ControllerBase
+    {
+        private readonly IBirdSightingDao birdSightingDao;
+        public BirdSightingController(IBirdSightingDao birdSightingDao)
+        {
+            this.birdSightingDao = birdSightingDao;
+
+        }
+
+        [HttpGet("/birds/{birdId}/notes")]
+        public List<BirdSighting> getNotes(int birdId)
+        {
+            List<BirdSighting> sightingList = birdSightingDao.getSightings(birdId);
+
+            if (sightingList == null)
+            {
+                Console.WriteLine("No bird");
+            }
+
+            return sightingList;
+
+        }
+
+        [HttpGet("/note/{id}")]
+        public BirdSighting getNote(int id)
+        {
+            BirdSighting sighting = birdSightingDao.getBirdSighting(id);
+
+            if (sighting == null)
+            {
+                Console.WriteLine("No bird");
+            }
+
+            return sighting;
+
+        }
+
+        [HttpPost("/newNote/{birdId}")]
```

Comment:
This route shouldn't include `{birdId}`.

Reply:
Done